README.md

```diff
@@ -1,64 +1,31 @@
-nickel.rs
-=======
-[![Build Status](https://travis-ci.org/nickel-org/nickel.rs.png?branch=master)](https://travis-ci.org/nickel-org/nickel.rs)
-[![](http://meritbadge.herokuapp.com/nickel)](https://crates.io/crates/nickel)
-[![Join the chat at https://gitter.im/nickel-org/nickel.rs](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/nickel-org/nickel.rs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+# [nickel.rs](http://nickel.rs) [![Build Status](https://travis-ci.org/nickel-org/nickel.rs.png?branch=master)](https://travis-ci.org/nickel-org/nickel.rs) [![](http://meritbadge.herokuapp.com/nickel)](https://crates.io/crates/nickel) [![Join the chat at https://gitter.im/nickel-org/nickel.rs](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/nickel-org/nickel.rs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-nickel is supposed to be a simple and lightweight foundation for web applications written in Rust. It's API is inspired by the popular express framework for JavaScript.
+[nickel.rs](http://nickel.rs) is a simple and lightweight foundation for web applications written in Rust. It's API is inspired by the popular express framework for JavaScript.
 
-Some of the features are:
+##Hello world
 
-* Easy handlers: A handler is just a function that takes a `&Request` and `&mut Response`
-* Variables in routes. Just write `my/route/:someid`
-* Easy parameter access: `request.param("someid")`
-* simple wildcard routes: `/some/*/route`
-* double wildcard routes: `/a/**/route`
-* middleware
-    * static file support
+### Dependencies
 
-##[Jump to the nickel.rs website](http://nickel.rs)
+You'll need to create a *Cargo.toml* that looks like this;
 
-#Getting started
-The easiest way to get started is to get the example running and play around with it. Let's do that real quick!
-
-##Clone the repository
-
-```shell
-git clone --recursive https://github.com/nickel-org/nickel.git
-```
-
-##Build nickel
-
-```shell
-cargo build --release
-```
-
-##Run the tests
-
-```shell
-cargo test
-```
-
-##Run the example
-
-```shell
-cargo run --example example
-```
-
-Then try `localhost:6767/user/4711` and `localhost:6767/bar`
-
-### Note about nightly
+```not_rust
+[package]
 
-To build on nightly, you will need to add `--features nightly` to the end of the above commands. Alternatively, if depending on the library you will need to add the following to your `Cargo.toml`.
+name = "my-nickel-app"
+version = "0.0.1"
+authors = ["yourname"]
 
-```not_rust
 [dependencies.nickel]
 version = "*"
 features = ["nightly"]
+
+[dependencies]
+rustc-serialize = "*"
```

Comment:
`rustc-serialize` was required for the old readme example before I trimmed it down, it's now only needed for some of the examples. It's probably worth adding some sort of explanation within the examples themselves (possibly comments beside the imports) or some comment similar to 'you can find the crates required to build the examples on crates.io'. Not sure which is best!